public/index.html

```diff
@@ -9,6 +9,12 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link
```

Comment:
why you are using boostrap?, this would be created from scratch, not using css libraries like boostrap, remove this